external_services/lti.py

```diff
@@ -8,6 +8,7 @@
 from oauthlib.oauth1 import Client, SIGNATURE_HMAC, SIGNATURE_TYPE_BODY, \
     SIGNATURE_TYPE_QUERY
 import json
+from urllib.parse import urlparse
```

Comment:
The other method, `urlsplit`, is a bit more preferred one.

> This is similar to urlparse(), but does not split the params from the URL. This should generally be used instead of urlparse() if --

- src: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlsplit

(You can see from [the source](https://github.com/python/cpython/blob/master/Lib/urllib/parse.py#L369) that urlparse uses urlsplit and just does more, which we don't need).